core/src/test/java/io/questdb/test/client/LineSenderBuilderTest.java

```diff
@@ -150,6 +188,67 @@ public void testBufferSizeDoubleSet() throws Exception {
+            assertConfStrError(Sender.builder().http(), "http::addr=localhost:8080;auto_flush_rows=100;auto_flush=off;", "protocol was already configured [protocol=http]");
+            assertConfStrError(Sender.builder().tcp(), "http::addr=localhost:8080;auto_flush_rows=100;auto_flush=off;", "protocol was already configured [protocol=tcp]");
+            assertConfStrError(Sender.builder().address("remote"), "http::addr=localhost:8080;auto_flush_rows=100;auto_flush=off;", "server address was already configured [address=remote]");
+            assertConfStrError(Sender.builder().address("remote:1234"), "http::addr=localhost:8080;auto_flush_rows=100;auto_flush=off;", "server address was already configured [address=remote]");
+            assertConfStrError(Sender.builder().port(1234), "http::addr=localhost:8080;auto_flush_rows=100;auto_flush=off;", "server port was already configured [port=1234]");
+            assertConfStrError(Sender.builder().advancedTls().disableCertificateValidation(), "http::addr=localhost:8080;tls_verify=on;", "tls_verify was already configured");
+            assertConfStrError(Sender.builder().enableTls(), "http::addr=localhost:8080;", "cannot use http protocol when TLS is enabled. use https instead");
+            assertConfStrError(Sender.builder().enableTls(), "tcp::addr=localhost:8080;", "cannot use tcp protocol when TLS is enabled. use tcps instead");
+
+            assertConfStrOk(Sender.builder().enableTls(), "https::addr=localhost:8080;");
+            assertConfStrOk("http", "addr=localhost:8080", "auto_flush_rows=100");
+            assertConfStrOk("http", "addr=localhost:8080", "auto_flush=on", "auto_flush_rows=100");
+            assertConfStrOk("http", "addr=localhost:8080", "auto_flush_rows=100", "auto_flush=on");
+            assertConfStrOk("http", "addr=localhost", "auto_flush=on");
+            assertConfStrOk("http::addr=localhost;auto_flush=off;");
+            assertConfStrOk("http::addr=localhost;");
+            assertConfStrOk("http::addr=localhost:8080;");
+            assertConfStrOk("http::addr=localhost:8080;token=foo;");
+            assertConfStrOk("http::addr=localhost:8080;token=foo=bar;");
+            assertConfStrOk("http", "addr=localhost:8080", "token=foo", "retry_timeout=1000", "max_buf_size=1000000");
+            assertConfStrOk("http", "addr=localhost:8080", "token=foo", "retry_timeout=1000", "max_buf_size=1000000");
+            assertConfStrOk("http::addr=localhost:8080;token=foo;max_buf_size=1000000;retry_timeout=1000;");
+            assertConfStrOk("https::addr=localhost:8080;tls_verify=unsafe_off;auto_flush_rows=100;");
+            assertConfStrOk("https::addr=localhost:8080;tls_verify=on;");
+            assertConfStrError("https::addr=2001:0db8:85a3:0000:0000:8a2e:0370:7334;tls_verify=on;", "cannot parse a port from the address. use IPv4 IP address or a domain name [address=2001:0db8:85a3:0000:0000:8a2e:0370:7334]");
```

Comment:
nit: `... address. use IPv4 IP address or a domain nam`
Should be no . or upper case Use ideally